datasets/sunbird/README.md

```diff
@@ -0,0 +1,142 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: https://github.com/huggingface/datasets-tagging
+---
+
+# Dataset Card for [Sunbird AI Parallel Text MT Dataset]
+
+## Table of Contents
+- [Dataset Card for [Dataset Name]](#dataset-card-for-dataset-name)
+  - [Table of Contents](#table-of-contents)
+  - [Dataset Description](#dataset-description)
+    - [Dataset Summary](#dataset-summary)
+    - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+    - [Languages](#languages)
+  - [Dataset Structure](#dataset-structure)
+    - [Data Instances](#data-instances)
+    - [Data Fields](#data-fields)
+    - [Data Splits](#data-splits)
+  - [Dataset Creation](#dataset-creation)
+    - [Curation Rationale](#curation-rationale)
+    - [Source Data](#source-data)
+      - [Initial Data Collection and Normalization](#initial-data-collection-and-normalization)
+      - [Who are the source language producers?](#who-are-the-source-language-producers)
+    - [Annotations](#annotations)
+      - [Annotation process](#annotation-process)
+      - [Who are the annotators?](#who-are-the-annotators)
+    - [Personal and Sensitive Information](#personal-and-sensitive-information)
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+    - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage: sunbird.ai**
+- **Repository: https://github.com/SunbirdAI/ug-language-parallel-text-dataset**
+- **Paper:**
+- **Leaderboard:**
+- **Point of Contact: sunbird.ai**
+
+### Dataset Summary
+
+This is Multi-way parallel text corpus of 5 key Ugandan languages intended for the task of machine translation
+
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+en-lug - English to Luganda
+en-run - English to Runyankore
+en-ach - English to Acholi
+en-teo - English to Itesot
+en-lgg - English to Lugbara
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
```

Comment:
Could you add a sample from the dataset here please ?

You can get one by using `load_dataset` and taking the first sample `dataset["train"][0]`